h/__main__.py

```diff
@@ -1,4 +1,4 @@
-from h.cli import main
+from h.cli import main  # pragma: no cover
 
-if __name__ == "__main__":
+if __name__ == "__main__":  # pragma: no cover
```

Comment:
I haven't found a place to put it to only need one.

Reply:
For a file like this I might just add it to the `omit` list in `.coveragerc`: https://github.com/hypothesis/h/blob/44ede4a2296f10c5a9d4e999cc26d32df19756b9/.coveragerc#L7-L10